server/views/referrals/courseParticipations/index.njk

```diff
@@ -2,6 +2,8 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 
+{%- from "moj/components/banner/macro.njk" import mojBanner -%}
```

Comment:
`{%-` -> `{%` (though I'm still not sure what the difference is 😅)

Looks like in the only other place we've imported an MoJ macro, we didn't line break after the GOV.UK ones (they're grouped as external imports), but this isn't a linter/formatter issue